README.md

```diff
@@ -0,0 +1,108 @@
+# Spring reCaptcha
+
+Library providing a custom filter to validate reCaptcha
+
+## Requirements
+
+* JDK 8 +
+* Spring Boot MVC application
+
+## Supported reCaptcha version
+
+* [X] reCaptcha v2
+* [ ] reCaptcha v3
+
+## Usage 
+
+To add the custom filter, you need to include the dependency and configure required property (see [Required configuration](#required-configuration)).
+
+```kotlin
+dependencies {
+  ...
+  implementation("com.ekino.oss.spring:spring-recaptcha:0.0.1-SNAPSHOT")
+  ...
+}
+```
+
+When calling an url requiring reCaptcha validation, you must send the reCatpcah response paramter, 
```

Comment:
small typo `reCaptcha` (it's not very pleasant to write 😛 ), and `parameter`

Reply:
I think I hate this word, 4 consonants in a row ... corrected btw

Reply:
the `reCaptcha` typo is still present 😅

Reply:
Sorry about that, I must have done mistake in my rebase process to keep a clean git history :s
Corrected now